<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DependencyWidget</class>
 <widget class="QWidget" name="DependencyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="twPackages">
     <column>
      <property name="text">
       <string>Package Name</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblStatusLine">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="installCommandLayout">
     <item>
      <widget class="QLabel" name="lblInstallCommand">
       <property name="text">
        <string>Install command:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leInstallCommand">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
